src/js/firstparties/twitter.js

```diff
@@ -63,13 +76,42 @@ function unwrapTwitterURLsInTimeline() {
   });
 
   // iterate another time to actually replace the link (of Twitter cards, e.g.)
-  findInAllFrames("a[href^='https://t.co/'], a[href^='http://t.co/'").forEach((link) => {
+  findInAllFrames(tcos_all).forEach((link) => {
     if (fixes.hasOwnProperty(link.href)) {
       unwrapTco(link, fixes[link.href]);
     }
   });
 }
 
+function unwrapTwitterPwaURLsInTimeline() {
+  // unwrap profile links
+  document.querySelectorAll(tcos_all).forEach((link) => {
+    // use cached value if possible
+    if (fixes.hasOwnProperty(link.href)) {
+      unwrapTco(link, fixes[link.href]);
+      return;
+    }
+
+    // find span element with link text in it
+    const allSpans = link.getElementsByTagName("span");
+    if (!allSpans) {
+      return;
+    }
+
+    const linkElement = link.getElementsByTagName("span")[0];
+    const elements = link_reg_ex.exec(linkElement.textContent);
```

Comment:
Getting errors (in Chrome) here, might be related to the empty `HTMLCollection` thing above:
```
twitter.js:102 Uncaught TypeError: Cannot read property 'textContent' of undefined
    at document.querySelectorAll.forEach (twitter.js:102)
    at NodeList.forEach (<anonymous>)
    at unwrapTwitterPwaURLsInTimeline (twitter.js:88)
```